### PR TITLE
Add contribution instructions for the Atlas CLI

### PR DESCRIPTION
**Pull request by ZachEddy**
# Contribution Instructions

If people want to CRUD features to or from the Atlas CLI, they now have a handy set of instructions in to help them out.

### CODE REVIEW

File: cli/atlas/README.md [hunk 1]
```diff
@@ -82,4 +82,40 @@ Of course, you may include all the flags in the `init-app` command.
...
\ No newline at end of file
+```
+
+## Contributing to the Atlas CLI
+Contributions to the Atlas CLI are welcome via pull requests and issues. If you're interested in making changes or adding new features, please take a minute to skim through these instructions.
+
+### Regenerating Template Bindata
+The `templates/` directory contains a set of Go templates. When the Atlas CLI bootstraps a new application, it uses these templates to render new application files.
+
+This project uses [go-bindata](https://github.com/jteeuwen/go-bindata) to package the Go templates and `atlas` source files together into a single binary. If your changes involve adding or updating templates, you will need regenerate the template bindata by running the following command:
```
Comment (johnbelamaric) [inline cli/atlas/README.md]: Why not a make target that does this and also the formatting?

File: cli/atlas/README.md [hunk 2]
```diff
@@ -82,4 +82,40 @@ Of course, you may include all the flags in the `init-app` command.
...
+
+This command regenerates the `templates/template-bindata.go` file. Your templating changes will take effect the next time you run the `atlas` binary. The go-bindata library does not generate correctly-formatted Go code, so please reformat `template-bindata.go` before opening a pull request.
+
+```
+# atlas-app-toolkit/cli/atlas
+go fmt templates/template-bindata.go
+```
+### Running the Integration Tests
+
+The Atlas CLI integration tests ensure that new changes do not break exists features. To run the Atlas CLI unit and integration tests locally, set `ec2=true` in your environment.
```
Comment (johnbelamaric) [inline cli/atlas/README.md]: s/ec2/e2e/

File: Makefile [hunk 3]
```diff
@@ -40,3 +40,6 @@ check-fmt:
 
 .PHONY: gen
 gen: .gen-op .gen-errdetails
+
+atlas-cli:
```
Comment (dmacthedestroyer) [inline Makefile]: should this be called `build-cli` or something? There's no verby thing to indicate to me that it does outside of reading the command

**Comment by ZachEddy** [comment]
@johnbelamaric Is this good to merge? I'd like to merge the open CLI-related PRs before migrating to the atlas-cli repository.

**Comment by dmacthedestroyer** [comment]
I'm going to close this since [atlas-cli](https://github.com/infobloxopen/atlas-cli) already exists. OK?

**Comment by ZachEddy** [comment]
Not if I close it first!

**Pull request closed without merging**